scrapers/cscd.py

```
@@ -0,0 +1,134 @@
+            
+            if len(links) == 1:
+                area = links[0].text
+            else:
+                area = links[1].text
+        elif h3.text == "Length":
+            length = h3.parent.find_all("div")[0].text
+        elif h3.text == "Difficulty":
+            raw_diff = h3.parent.find_all("div")[0].text.strip()
+            try:
+                difficulty = raw_diff.split("(")[1].split(")")[0]
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)